docs/modules/configuration/pages/jet-configuration.adoc

```diff
@@ -297,6 +297,18 @@ The most important properties are listed here:
 Each job has job-specific configuration options. These are covered
 in detail in xref:pipelines:configuring-jobs.adoc[].
 
+=== Isolated Jobs
```

Comment:
We need to consider the terms we want to present to the users. `Isolated Jobs` are specific to our internal discussions; user-facing API don't contain any mention of  `Isolated Jobs`

Reply:
I've replaced all with job placement